src/main/scala/com.snowplowanalytics.iglu/ctl/Server.scala

```diff
@@ -146,7 +148,7 @@ object Server {
     * @param writeKey temporary apikey allowed to write any Schema
     * @return HTTP POST-request ready to be sent
     */
-  def buildRequest(registryRoot: HttpUrl, isPublic: Boolean, schema: IgluSchema, writeKey: String): HttpRequest =
+  def buildRequest(registryRoot: HttpUrl, isPublic: Boolean, schema: SelfDescribingSchema[JValue], writeKey: String): HttpRequest =
```

Comment:
Do you think it is possible to get rid of json4s entirely in this release?

Reply:
I take a look at a little bit what we can do it for it. I think we need to update Iglu Client too in order to get rid of json4s. If you think it is okay, I think it is feasible.

Reply:
That's totally okay and actually even preferrable.